src/renderers/shaders/ShaderLib/sprite_vert.glsl.js

```diff
@@ -20,9 +20,7 @@ void main() {
 
 	#ifndef USE_SIZEATTENUATION
 
-		bool isPerspective = isPerspectiveMatrix( projectionMatrix );
-
-		if ( isPerspective ) scale *= - mvPosition.z;
+		// if ( ! isOrthographic ) scale *= - mvPosition.z;
```

Comment:
Why is this line commented out?

Reply:
Just pushed a fix, noticed just before you commented.